src/main/java/io/confluent/connect/hdfs/avro/AvroRecordWriterProvider.java

```diff
@@ -56,6 +59,7 @@ public String getExtension() {
     return new RecordWriter<SinkRecord>(){
       @Override
       public void write(SinkRecord record) throws IOException {
+        log.debug("Sink record: {}", record.toString());
```

Comment:
I have a follow up patch that removes this anyway, but this results in *very large* log files. That's fine when it's useful for tests, at a low enough log level, and can be isolated from normal logs, but this was a bit excessive :)